Tools/ardupilotwaf/px4.py

```diff
@@ -238,7 +242,7 @@ def bldpath(path):
     # stop using the make-based build system
     env.PX4_ROMFS_SRC = 'mk/PX4/ROMFS'
     env.PX4_ROMFS_BLD = 'px4-extra-files/ROMFS'
-    env.PX4_BOOTLOADER = 'mk/PX4/bootloader/%s' % env.PX4_BOOTLOADER_NAME
+    env.PX4_BOOTLOADER = '/../bootloader/%s' % env.PX4_BOOTLOADER_NAME
```

Comment:
Yep... That's just to gurantee I don't end up with 'previous/dir../bootloader' instead of 'previous/dir/../bootloader'.